vcd/resource_vcd_vdc_group.go

```diff
@@ -0,0 +1,464 @@
+		},
+		"status": {
+			Type:     schema.TypeString,
+			Computed: true,
+			Description: "The status that the VDC can be in e.g. 'SAVING', 'SAVED', 'CONFIGURING'," +
+				" 'REALIZED', 'REALIZATION_FAILED', 'DELETING', 'DELETE_FAILED', 'OBJECT_NOT_FOUND'," +
+				" 'UNCONFIGURED').",
+		},
+		"org_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Organization VDC belongs",
+		},
+		"org_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Organization VDC belongs",
+		},
+		"site_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Site VDC belongs",
+		},
+		"site_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Site VDC belongs",
+		},
+		"vdc_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "VDC name",
+		},
+		"vdc_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "VDC ID",
+		},
+	},
+}
+
+func resourceVdcGroup() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   resourceVcdVdcGroupRead,
+		CreateContext: resourceVcdVdcGroupCreate,
+		UpdateContext: resourceVcdVdcGroupUpdate,
+		DeleteContext: resourceVcdVdcGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVdcGroupImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of VDC group",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "VDC group description",
+			},
+			"dfw_enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Distributed firewall status",
+			},
+			"starting_vdc_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Starting VDC Id",
+			},
+			"participating_vdc_ids": {
+				Type:        schema.TypeSet,
+				Required:    true,
+				Description: "Participating VCD IDs",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"default_policy_status": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Default Policy Status",
+			},
+			"error_message": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "More detailed error message when VDC group has error status",
+			},
+			"local_egress": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Status whether local egress is enabled for a universal router belonging to a universal VDC group.",
```

Comment:
I think we don't use dots at the end of `Description`, so let's make this PR consistent:

```suggestion
				Description: "Status whether local egress is enabled for a universal router belonging to a universal VDC group",
```

Reply:
removed